privacy-policy.php

```diff
@@ -0,0 +1,45 @@
+<?php
+    include '_templates/sitewide.php';
+    $page['title'] = 'Privacy Policy';
+    include '_templates/header.php';
+?>
+            <div class="row" style="max-width:1000px;">
+                <h1>Privacy Policy</h1>
+                <h2>Do Not Track</h2>
+                <p>This site honors the do-not-track setting found in most modern browsers, and disables Google Universal Analytics when it is present. Unfortunately, we cannot remove other tracking methods without severely impacting the use of the site. Should you wish to do so, reference the section on disabling cookies.</p>
+                <p>Below you will find links to enabling DNT in your browser.</p>
+                <ul>
+                    <li><a target="_blank" href="https://support.google.com/chrome/answer/114836">Google Chrome</a></li>
+                    <li><a target="_blank" href="https://www.mozilla.org/en-US/firefox/dnt/">Mozilla Firefox</a></li>
+                    <li><a target="_blank" href="http://windows.microsoft.com/en-GB/internet-explorer/use-tracking-protection">Microsoft Internet Explorer</a></li>
+                    <li><a target="_blank" href="http://help.opera.com/Windows/12.10/en/notrack.html">Opera</a></li>
+                </ul>
```

Comment:
I really don't think we need to include the above section. If they know what DNT is or already have it enabled, fine. But it's not our job to educate people on how to enable it.